src/resources/ble_esp32.cc

```diff
@@ -414,10 +409,6 @@ int BLEResourceGroup::init_server() {
     }
 
     ble_hs_cfg.reset_cb = ble_on_reset;
-    // Start the host thread.
-    if (!_server_config->spawn(NIMBLE_STACK_SIZE)) {
```

Comment:
Is the NIMBLE_STACK_SIZE constant now unused? Should it be passed to the BLEEventSource thread constructor?

Reply:
Yes. Good point.